scripts/release/setup-workspace

```diff
@@ -40,79 +21,73 @@ abspath () {
     echo "$_path"
 }
 
+if [ "$1" = "--help" ]; then
+    usage
+    exit 1
+fi
+
 WORKSPACEDIR=$PWD/workspace
-saved=
 manifest=
-extra_disabled=0
+extra_disabled=
```

Comment:
@kergoth, this is causing
```
$ source setup-mel -w 123 -x manifests/extra/meta-mentor-industrial/mel-2018.9.29-meta-mentor-industrial.manifest -X
/data/mgc-uno/embedded/mel/2018.9.29/scripts/setup-workspace: line 66: [: : integer expression expected
Error: -x and -X are mutually exclusive
```
pointing out the error seen on line 66 as we use integer test there.

Reply:
Ah, missed one. Thanks, I'll fix it.